README.md

```diff
@@ -1 +1,138 @@
-# sparkconnect
\ No newline at end of file
+# spark_connect_labextension
```

Comment:
What was the original one?

Reply:
The one that is in the name of this repo, `sparkconnect` :) Or something as simple.

Reply:
I'm afraid that `sparkconnect` would suggest that this repo is the Spark Connect server as opposed to a JupyterLab Extension that integrates Spark Connect, but if you have a simpler name that is not ambiguous we can consider renaming it I suppose

Reply:
When this extension replaces the current Spark connector, it will just get that name. So I don't worry that much about the current name, in the end it will be called Spark connector.

Reply:
But perhaps you can already call it precisely that, `sparkconnector` ?

Reply:
Can we update the name then?

Reply:
Will do